include/maliput/common/logger.h

```diff
@@ -4,7 +4,7 @@
 /// To log with `maliput::common::logger`, you should:
 ///
 /// <pre>
-///   maliput::log()->trace("Trace message: {} {}", something, some_other);
+///   maliput::log()->trace("The approximate value is ", 3.14159);
```

Comment:
If we agree in this format then I will proceed and change all the downstream ocurrences

Reply:
@agalbachicar